### DISCUSSION TITLE
NeoVim port of the VSCode plugin

### DISCUSSION THREAD

**Discussion by dkbast** [serverpod/serverpod#2061] [Show and tell]
Hey there :) - I ported the VSCode plugin to coc.vim which is a plugin host for vscode like extensions in vim. Happy to open a PR if that is something that would be of interest to the team? I also didn't upload it to npm yet, which would make it possible to install it from within coc without referencing the repo:
https://github.com/dkbast/coc-serverpod

**Comment by SandPod** [comment]
Hey!

That is awesome! 🙌 

Currently, the team will not have the bandwidth to maintain it. Therefore, we propose that you can publish it under your name, and then we can link to it from our documentation.

Does that sound good to you?
